package-lock.json

```diff
@@ -2472,7 +2466,7 @@
         "gulp-cli": {
           "version": "2.0.1",
           "resolved": "https://registry.npmjs.org/gulp-cli/-/gulp-cli-2.0.1.tgz",
-          "integrity": "sha512-RxujJJdN8/O6IW2nPugl7YazhmrIEjmiVfPKrWt68r71UCaLKS71Hp0gpKT+F6qOUFtr7KqtifDKaAJPRVvMYQ==",
+          "integrity": "sha1-eEfiIMs2YvK+im1XK/FOF75amUs=",
```

Comment:
It's a shame these change from 512 to 1.  Would be nice to get to the bottom of this as it really isn't related to changes in this PR.

Reply:
Yeah, I wish I knew what caused this. Seems random, but it’s meant to be entirely deterministic?